builtin-functions/kphp-full/_functions.txt

```diff
@@ -141,6 +141,7 @@ function ob_get_flush () ::: string | false;
 function ob_get_length () ::: int | false;
 function ob_get_level () ::: int;
 
+function headers_sent () ::: bool;
```

Comment:
@kojidev 
> well I aimed to add a minimal support for headers_sent(), not a full support.

We no need minimal support. We want to get a compatible with PHP implementation.

> We have on_start that does not compile with 2nd parameter passed, when it should, because it would in php:

It's not cool 

>  Do we really want to add the full support, or even ignore the parameters when we don't have a used case for that but we do for just a function call?

Yes, we can ignore parameters and, for example, throw a warning or error (need to decide what is better) in runtime if parameters are not null.

Reply:
You tell me what to throw an error or a warning I'll do it, but I still don't see how ignoring / or throwing unexpected errors (they are not thrown in php) is more compatible than just refuse to compile certain cases

Reply:
@kojidev 
First of all, KPHP is a subset of PHP. The following code:
```
headers_sent($filename, $linenum)
```
can be supported in the subset.

The concept of the subset means that almost any `KPHP` program (including those that use the standard library) should be a valid program in PHP. To maximize the subset, the code above must be compilable by the KPHP compiler. The semantics of `headers_sent` do not contradict the principles of `KPHP` and can potentially be implemented in `KPHP`.

I agree that the current implementation of dumping the filename and line where headers have been sent is difficult and time-consuming. Therefore, we suggest temporarily omitting this case and finding a more balanced solution by considering the following points:
* Ignore parameters 
* Throw a warning if params are not null
* Always set default values (e.g. empty string and 0)

Reply:
added arguments, they are ignored